pkg/jsonnet/eval.go

```diff
@@ -75,32 +79,68 @@ func (o Opts) Clone() Opts {
 	}
 }
 
-// MakeVM returns a Jsonnet VM with some extensions of Tanka, including:
+type vmPool struct {
+	mutex     sync.Mutex
+	available map[string][]*jsonnet.VM
+}
+
+var VMPool = vmPool{
+	available: map[string][]*jsonnet.VM{},
+}
+
+func optsHash(opts Opts) string {
```

Comment:
I wouldn't do the `hash()` function because it's not an actual full hash. It's only used for the VMPool, it doesn't have all attributes, only the ones that require a new VM